tests/acceptance/home/edit-observation/edit-observation-test.js

```diff
@@ -56,9 +56,9 @@ module('Acceptance | edit-observation', function (hooks) {
     await visit('/edit-observation/1');
     assert.strictEqual(currentURL(), '/edit-observation/1');
 
-    assert.dom('[data-test-input-edit-obser-pikaday]').exists();
     assert
       .dom('[data-test-input-edit-obser-pikaday]')
+      .exists()
       .hasValue(
         moment(this.currentObservation.observationDate).format('YYYY-MM-DD')
```

Comment:
My previous comment was just reminder that you can link functions and `exists()` is not necessary there but you cal leave it as it is